packages/gatsby/src/utils/webpack.config.js

```diff
@@ -271,6 +271,13 @@ module.exports = async (
         use: [{
           loader: require.resolve(`./reach-router-add-basecontext-export-loader`),
         }],
+      },
+      {
+        test: (input) => {
```

Comment:
@TylerBarnes I think we only want to add this in develop mode so when `stage === 'develop'`

Reply:
updated this, thanks @wardpeet !